release-please-config.json

```diff
@@ -12,11 +13,11 @@
     "packages/hackathon": { },
     "packages/run-it": { "release-as": "" },
     "packages/sdk": { },
-    "packages/sdk-codegen": { "release-as": "21.0.20" },
-    "packages/sdk-codegen-scripts": { "release-as": "21.0.21" },
-    "packages/sdk-codegen-utils": { "release-as": "21.0.13"},
+    "packages/sdk-codegen": { "release-as": "21.0.21" },
+    "packages/sdk-codegen-scripts": { "release-as": "21.0.22" },
+    "packages/sdk-codegen-utils": { "release-as": "21.0.12"},
```

Comment:
current version is `21.0.11`, not sure why i'd previously set the next version to be `21.0.13`